packages/common/utils/src/mapValues.ts

```diff
@@ -11,8 +13,11 @@
  * mapValues({ foo: 1, bar: 2 }, x => x * 2)
  * // => { foo: 2, bar: 4 }
  */
-export function mapValues<T, U>(value: T, mapper: (value: T[keyof T]) => U): { [K in keyof T]: U } {
-  const entries = Object.entries(value);
+export function mapValues<T extends Record<PropertyKey, T[ObjectKeys<T>]>, U>(
+  value: T,
+  mapper: (value: T[keyof T]) => U
+): { [K in keyof T]: U } {
+  const entries = Object.entries(value) as Array<[ObjectKeys<T>, T[ObjectKeys<T>]]>;
```

Comment:
This assertion refers to exactly the same type of the result of `objectEntries`.
So I think you can import and just use it.

```typescript
import { objectEntries } from './object';

const entries = objectEntries(value); // Array<[ObjectKeys<T>, T[ObjectKeys<T>]]>
```

But I think any type assertions are not required here. This function is enough to be clear and type-safe without it.

Reply:
> But I think any type assertions are not required here. This function is enough to be clear and type-safe without it.

Yes, that's right. In this case type assertion doesn't need. I removed type assertion. 

When you find  any other problems in this PR, please tell me again.